Scripts/fastlane/helpers/ios_git_helper.rb

```diff
@@ -73,7 +73,7 @@ def self.final_tag(version)
 
       def self.localize_project()
         Action.sh("cd .. && ./Scripts/localize.py")
-        Action.sh("cd .. && git add ./WordPress/Resources/.")
+        Action.sh("git add WordPress/*.lproj/Localizable.strings")
```

Comment:
@jtreanor I think this should be `../WordPress/*.lproj/Localizable.strings` because the script runs from the `/Scripts` folder.

Reply:
Well spotted. I'll test this out and post back here.